2021-09-16 23:08:11,688 INFO    MainThread:29740 [wandb_setup.py:_flush():69] setting env: {}
2021-09-16 23:08:11,688 INFO    MainThread:29740 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-16 23:08:11,688 INFO    MainThread:29740 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20210916_230811-dd9e1jwm/logs/debug.log
2021-09-16 23:08:11,688 INFO    MainThread:29740 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20210916_230811-dd9e1jwm/logs/debug-internal.log
2021-09-16 23:08:11,689 INFO    MainThread:29740 [wandb_init.py:init():381] calling init triggers
2021-09-16 23:08:11,689 INFO    MainThread:29740 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-09-16 23:08:11,689 INFO    MainThread:29740 [wandb_init.py:init():430] starting backend
2021-09-16 23:08:11,689 INFO    MainThread:29740 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-16 23:08:11,697 INFO    MainThread:29740 [backend.py:ensure_launched():135] starting backend process...
2021-09-16 23:08:11,709 INFO    MainThread:29740 [backend.py:ensure_launched():140] started backend process with pid: 29848
2021-09-16 23:08:11,711 INFO    MainThread:29740 [wandb_init.py:init():435] backend started and connected
2021-09-16 23:08:11,713 INFO    MainThread:29740 [wandb_init.py:init():494] updated telemetry
2021-09-16 23:08:11,713 INFO    MainThread:29740 [wandb_init.py:init():517] communicating current version
2021-09-16 23:08:12,571 INFO    MainThread:29740 [wandb_init.py:init():522] got version response 
2021-09-16 23:08:12,571 INFO    MainThread:29740 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-16 23:08:12,729 INFO    MainThread:29740 [wandb_init.py:init():557] starting run threads in backend
2021-09-16 23:08:17,739 INFO    MainThread:29740 [wandb_run.py:_console_start():1605] atexit reg
2021-09-16 23:08:17,740 INFO    MainThread:29740 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-16 23:08:17,740 INFO    MainThread:29740 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-16 23:08:17,744 INFO    MainThread:29740 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-16 23:08:17,744 INFO    MainThread:29740 [wandb_init.py:init():582] run started, returning control to user process
2021-09-16 23:08:25,603 INFO    MainThread:29740 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 1
2021-09-16 23:08:25,605 INFO    MainThread:29740 [wandb_run.py:_restore():1547] restore
2021-09-16 23:08:27,842 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-16 23:08:28,244 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-16 23:08:28,365 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-16 23:08:28,778 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 10062
}

2021-09-16 23:08:28,883 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 10471
}

2021-09-16 23:08:28,986 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10471
  total_bytes: 10471
}

2021-09-16 23:08:29,089 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10471
  total_bytes: 10471
}

2021-09-16 23:08:29,192 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10471
  total_bytes: 10471
}

2021-09-16 23:08:29,295 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10471
  total_bytes: 10471
}

2021-09-16 23:08:29,543 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10471
  total_bytes: 10471
}

2021-09-16 23:08:29,830 INFO    MainThread:29740 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10471
  total_bytes: 10471
}
local_info {
}

2021-09-16 23:08:31,156 INFO    MainThread:29740 [wandb_run.py:_show_files():1947] logging synced files
